src/main/java/org/spongepowered/common/data/DataRegistrar.java

```diff
@@ -590,6 +590,8 @@ public static void setupSerialization(Game game) {
         dataManager.registerDataProcessorAndImpl(DyeableData.class, SpongeDyeableData.class, ImmutableDyeableData.class,
                 ImmutableSpongeDyeableData.class, dyeableDataProcessor);
 
+        dataManager.registerDataProcessorAndImpl(FuseData.class, SpongeFuseData.class, ImmutableFuseData.class, ImmutableSpongeFuseData.class, 
```

Comment:
Use `registerDualProcessor`.

This is the wrong section for entity-related processors